src/atoms/Text/__tests__/text.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+
+import Text from 'atoms/Text';
+
+describe('<Text />', () => {
+
+  it('renders', () => {
+    const wrapper = shallow(<Text />);
+
+    expect(wrapper.length).to.equal(1);
```

Comment:
`type` doesn't work like that. In most cases type will be `<div>`, the wrapping element.  If you had a component that returns `<Something>thing</Something>` its type will be `Something`.